tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/json/JSONTest.java

```diff
@@ -118,6 +118,11 @@ public void testParseNull() {
         Assert.assertNull(returns[0]);
     }
 
+    @Test(description = "Get JSON with null values")
```

Comment:
How is this test relevant to this fix? This works already also.

Reply:
This test was added since to check weather `null` and `()` is working correctly in JSON context after changing the `toString` method. Since now that changed is not included in this PR anymore, I will remove this